.github/CHANGELOG.md

```diff
@@ -6,9 +6,10 @@
  
  Note each MPI process will return the overall result of the adjoint method. The MPI adjoint method has two options:
  1. Default method is faster but requires more memory. With the default method, a separate `bra` is created for each observable and `ket` is only updated once for each operation, regardless of the number of observables. This approach may consume more memory due to the creation of multiple `bra`s.  2. Memory-optimized method requires less memory but is slower. The memory-optimized method uses a single `bra` object that is reused for all observables. The `ket` needs to be updated `n` times, where `n` is the number of observables, for each operation. This approach reduces memory consumption as only one bra object is created. However, it may lead to slower execution due to the multiple `ket` updates per gate operation.
```

Comment:
```suggestion
 1. The default method is faster if the available problem fits into GPU memory, and will simply enabled with the `mpi=True` device argument. With the default method, a separate `bra` is created for each observable and the `ket` is only updated once for each operation, regardless of the number of observables. This approach may consume more memory due to the up-front creation of multiple `bra`s.  
 2. The memory-optimized method requires less memory but is slower due serialization of the execution. The memory-optimized method uses a single `bra` object that is reused for all observables. The `ket` needs to be updated `n` times, where `n` is the number of observables, for each operation. This approach reduces memory consumption as only one `bra` object is created. However, it may lead to slower execution due to the multiple `ket` updates per gate operation.
```

Reply:
Hopefully the above does not affect the doc. If it does, it may be best to indent the enumerated entries until they appear under the appropriate changelog entry.

Reply:
Updated.